lib/_emerge/actions.py

```diff
@@ -3165,8 +3165,12 @@ def run_action(emerge_config):
 	disable_emergelog = False
 	for x in ("--pretend", "--fetchonly", "--fetch-all-uri"):
 		if x in emerge_config.opts:
-			disable_emergelog = True
-			break
+			if x == "--fetchonly" and "--quiet" in emerge_config.opts:
+				# Log will be used to store fetch progress
```

Comment:
You'll have to check for write access to emerge.log here, just as you did in e26a08896f098ff93530efcd2d77d60d2eff0257.

Reply:
Have added the access check now.

Reply:
Thanks!